_dasldemos/fh-cbio-intro.md

```diff
@@ -0,0 +1,247 @@
+| Generic Assays: SBS Mutational Signature     | Required (if including this type of data)        | data_mutational_signature_contribution_SBS.txt | Tab Separated Value (TSV)          | A data file describing the contribution of each mutational signature (values should range from 0 to 1) | [Readme for SBS Mutational Signature](https://github.com/FredHutch/tg-cbioportal-data/01_file_formats/08_generic_assay_mutational_signature) | [Example of data_mutational_signature_contribution_SBS.txt](https://github.com/FredHutch/tg-cbioportal-data/01_file_formats/08_generic_assay_mutational_signature/data_mutational_signature_contribution_SBS.txt)
+
+
+## Publicly available tools for data formatting
+There are many publicly available formatting tools that may help with the formatting process. When deciding which one works best for you, it ultimately depends on what tools you're comfortable with and what kinds of data you're uploading, but here are a few options that might help get you started:
+
+| Tool Name                                                                                          | Description                                                                             | Advantages                                                                                          | Disadvantages                                                                                                           | FH Repository Link                                                                                     |
+|----------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------|
+| [Data-processor](https://github.com/MJKorte/Data-processor)                                     | Formats clinical data tables in multi-tab Excel files to cBioportal format             | - Useful for varied clinical data fields.<br>- Supports multi-tab Excel files.<br>- Easy terminal execution | - Does not seem to work to generate clinical data files.<br>- Requires adherence to specific clinical data variable names from [cBioPortal Clinical Data Dictionary] | [Data_processor](https://github.com/FredHutch/tg-cbioportal-data/05_useful_tools/01_Data_processor) |
```

Comment:
the links to `FredHutch/tg-cbioportal-data` I think are not formatted correctly. I realize the work being referred to is on a different branch, but even if it was on main, these URLs wouldn't work. e.g., I think 

`https://github.com/FredHutch/tg-cbioportal-data/05_useful_tools/01_Data_processor`

would instead be like 

`https://github.com/FredHutch/tg-cbioportal-data/tree/main/05_useful_tools/01_Data_processor`

Reply:
Damn, great catch. Merged the `tg-cbioportal-data` PR and updated to the formatting of your second link. Built the site and links work now. Thanks @sckott !